lib/matplotlib/collections.py

```diff
@@ -2200,6 +2200,9 @@ def get_cursor_data(self, event):
         if len(info["ind"]) == 1:
             ind, = info["ind"]
             array = self.get_array()
-            return array[ind] if array else None
+            if array is not None:
+                return array[ind]
+            else:
+                return None
```

Comment:
Slight preference for keeping the ternary expression:
```suggestion
            return array[ind] if array is not None else None
```